modules/installation-configuration-parameters.adoc

```diff
@@ -1743,6 +1743,58 @@ Supplying more than one user-assigned identity is an experimental feature, which
 
 *Value:* `VMGuestStateOnly` is the only supported value.
 
+|controlPlane:
+  diskSetup:
+|Specifies node component information for dedicated disk configuration.
+
+*Value:* Array of objects. Each object includes a `type` and `etcd` as described in the following rows of this table.
```

Comment:
```suggestion
*Value:* Array of objects. Each object includes the `type` and `etcd` values as described in the following rows of this table.
```

Assuming `type` and `etcd` are values. Please replace `values` with suitable terms such as fields, paarameters etc